core/src/main/scala/org/apache/spark/sql/streaming/SnappySinkCallback.scala

```diff
@@ -148,7 +148,6 @@ import org.apache.spark.sql.snappy._
 class DefaultSnappySinkCallback extends SnappySinkCallback {
```

Comment:
hashAggregateSizeChanged name at line 111 is confusing. Should be "hashAggregateSizeIsDefault"